src/prefect/runner/runner.py

```diff
@@ -374,6 +373,8 @@ def goodbye_flow(name):
         webserver = webserver if webserver is not None else self.webserver
 
         if webserver or PREFECT_RUNNER_SERVER_ENABLE.value():
+            from prefect.runner.server import start_webserver
+
```

Comment:
I think moving this here may be causing the failing tests but haven't checked

Reply:
Hope you don't mind, I'd love to get this in so I made the fix for these failing tests in [d65ac43](https://github.com/PrefectHQ/prefect/pull/13281/commits/d65ac430a62c9c6e20f9c62e9d59d5b05f93832a)